packages/botonic-plugin-knowledge-bases/src/index.ts

```diff
@@ -0,0 +1,50 @@
+import { HubtypeSession, Plugin, PluginPreRequest } from '@botonic/core'
+
+import { HubtypeApiService } from './hubtype-knowledge-api-service'
+import { KnowledgebaseResponse, PluginKnowledgeBaseOptions } from './types'
+
+const isProd = process.env.NODE_ENV === 'production'
+
+export default class BotonicPluginKnowledgeBases implements Plugin {
+  private readonly apiService: HubtypeApiService
+  private readonly authToken: string
+
+  constructor(options: PluginKnowledgeBaseOptions) {
+    this.apiService = new HubtypeApiService(
+      options.knowledgeBaseId,
+      options.host,
+      options.timeout
+    )
+    this.authToken = options.authToken || ''
+  }
+
+  async pre(_request: PluginPreRequest): Promise<void> {
```

Comment:
with latests updates of Plugin interface no need to define empty `pre` or `post` calls